vsdownload.py

```diff
@@ -175,7 +206,6 @@ def setPackageSelection(args, packages):
                     args.package.append(key)
                 else:
                     args.ignore.append(key)
-        p = packages[key][0]
```

Comment:
This looks unrelated?

As general commentery, I recently tried to use the `--sdk-version` option, and I think we should error out, if the option was specified but it didn't match any packages - as it's not entirely obvious in which form the version number needs to be passed (e.g. `10.0.17763` without the trailing `.0`). I could try to do such a PR separately.

Reply:
Actually, when poking around more here - I see that it seems like this line is totally unused and broken leftover from whenever I first implemented this, so we probably just should remove it, independently from these other changes.